plugin/trino-hive-hadoop2/src/test/java/io/trino/plugin/hive/TestHive.java

```diff
@@ -27,6 +29,7 @@
 public class TestHive
         extends AbstractTestHive
 {
+    protected static final PartitionStatistics EMPTY_TABLE_STATISTICS = PartitionStatistics.empty();
```

Comment:
I think you can use `PartitionStatistics.empty()` directly instead of this variable